src/templates/chatbox_head.html

```diff
@@ -18,7 +18,6 @@
         </div>
     </div>
     <div class="chatbox-buttons row no-gutters">
-        <a class="chatbox-btn close-chatbox-button fa fa-times" title="{{{o.info_close}}}"></a>
-        <a class="chatbox-btn show-user-details-modal fa fa-id-card" title="{{{o.info_details}}}"></a>
+        <!----a class="chatbox-btn show-user-details-modal fa fa-id-card" title="{{{o.info_details}}}"--><!----/a-->
```

Comment:
Please re-add the close button, see the discussion in #1611.

Also, don't comment out markup, delete it entirely if it's no longer used.